.github/workflows/summary.yml

```
@@ -4,31 +4,140 @@
 
-      - name: Run AI inference
-        id: inference
-        uses: actions/ai-inference@v1
-        with:
-          prompt: |
-            Summarize the following GitHub issue in one paragraph:
-            Title: ${{ github.event.issue.title }}
-            Body: ${{ github.event.issue.body }}
+      - name: Generate issue summary
+        id: summary
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:27:46: Double quote to prevent globbing and word splitting

(shellcheck)